subprojects/configuration-cache-report/src/main/kotlin/ConfigurationCacheReportPage.kt

```diff
@@ -176,6 +177,13 @@ object ConfigurationCacheReportPage : Component<ConfigurationCacheReportPage.Mod
         )
     )
 
+    private
+    fun Model.summary() =
+        "$totalProblems problems were found $cacheAction the configuration cache".let {
```

Comment:
:o: Opportunistic comment: it would be nice to handle single problem report with a singular text as done for the console at https://github.com/gradle/gradle/pull/17629/files#diff-fced8a783858536ab60fb1feb75810f72a39ae053ae80690430ebf8400d48f5dR110

Reply:
Right! Nice catch!

Reply:
✅ Done

> ⭕ Opportunistic comment: it would be nice to handle single problem report with a singular text as done for the console at https://github.com/gradle/gradle/pull/17629/files#diff-fced8a783858536ab60fb1feb75810f72a39ae053ae80690430ebf8400d48f5dR110